worker/mails/mail.go

```diff
@@ -88,6 +98,17 @@ func SendMail(ctx *job.TaskContext) error {
 			opts.ReplyTo = &mail.Address{Name: name, Email: replyTo}
 		}
 		opts.RecipientName = toAddr.Name
+	case mail.ModeCampaign:
```

Comment:
ModeFromStack case and ModeCampaign cas seems to be exactly the same (unless I miss something).

Can't golang group similar cases like we can do in C?

probably something like 

```golang
case mail.ModeFromStack, mail.ModeCampaign:
```

This would avoid code duplication.

Reply:
Good point. They were not the same at first and I didn't think about grouping them afterwards.